README.md

```diff
@@ -56,7 +57,7 @@ Give an example
 
 ## Deployment
 
-Add additional notes about how to deploy this on a live system
+We are using Travis for continous integration and Heroku for deployment.
```

Comment:
Why on earth are you including the README updates in this PR? Please remove this.